<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWgtZhouShi</class>
 <widget class="QWidget" name="QWgtZhouShi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="ZsKjgWgt" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>900</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="ZsHwWgt" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>902</y>
     <width>450</width>
     <height>256</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="PtzWgt" native="true">
   <property name="geometry">
    <rect>
     <x>1144</x>
     <y>900</y>
     <width>456</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(152, 152, 152);</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>方位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>转速：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>俯仰：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pitch">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_orientation">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_mid">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>中速</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_high">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>高速</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_low">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>低速</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/round.png);</string>
    </property>
    <widget class="QPushButton" name="btn_right">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>28</width>
       <height>55</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-image: url(:/resource/btn_right_hover.png);
background:transparent;
}
QPushButton:pressed
{	
	border-image: url(:/resource/btn_right_press.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_down">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>55</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-image: url(:/resource/btn_down_hover.png);
background:transparent;
}
QPushButton:pressed
{	
	border-image: url(:/resource/btn_down_press.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_left">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>28</width>
       <height>55</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-image: url(:/resource/btn_left_hover.png);
background:transparent;
}
QPushButton:pressed
{	
	border-image: url(:/resource/btn_left_press.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_up">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>55</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-image: url(:/resource/btn_up_hover.png);
background:transparent;
}
QPushButton:pressed
{	
	border-image: url(:/resource/btn_up_press.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TitileWgt_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>456</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(76,76,76);</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(225, 225, 225);</string>
     </property>
     <property name="text">
      <string>云台控制</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BtnVideoChange">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#BtnVideoChange
{
	color:rgb(0,0,0);
	border-image: url(:/resource/button1.png);
}</string>
    </property>
    <property name="text">
     <string>切换</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_pitch</zorder>
   <zorder>label_orientation</zorder>
   <zorder>label_speed</zorder>
   <zorder>radio_mid</zorder>
   <zorder>radio_high</zorder>
   <zorder>radio_low</zorder>
   <zorder>TitileWgt_2</zorder>
   <zorder>BtnVideoChange</zorder>
  </widget>
  <widget class="QPushButton" name="BtnZoomAdd">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>910</y>
     <width>82</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#BtnZoomAdd
{
	color:rgb(0,0,0);
	border-image: url(:/resource/button1.png);
}</string>
   </property>
   <property name="text">
    <string>变倍 +</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnZoomSub">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>950</y>
     <width>82</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnZoomSub
{
	color:rgb(0,0,0);
	border-image: url(:/resource/button1.png);
}</string>
   </property>
   <property name="text">
    <string>变倍 -</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnFocusAdd">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>990</y>
     <width>82</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#BtnFocusAdd
{
	color:rgb(0,0,0);
	border-image: url(:/resource/button1.png);
}</string>
   </property>
   <property name="text">
    <string>焦距 +</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnFocusSub">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>1030</y>
     <width>82</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#BtnFocusSub
{
	color:rgb(0,0,0);
	border-image: url(:/resource/button1.png);
}</string>
   </property>
   <property name="text">
    <string>焦距 -</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>900</y>
     <width>701</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(152, 152, 152);</string>
   </property>
   <widget class="QPushButton" name="btnAutoLight">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnAutoLight
{
	color:rgb(0,0,0);
	border-image: url(:/resource/button1.png);
}
</string>
    </property>
    <property name="text">
     <string>自动补光</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnManualLight">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnManualLight
{
	color:rgb(0,0,0);
	border-image: url(:/resource/button1.png);
}</string>
    </property>
    <property name="text">
     <string>手动补光</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>900</y>
     <width>1600</width>
     <height>2</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1158</y>
     <width>1600</width>
     <height>2</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>ZsKjgWgt</zorder>
  <zorder>ZsHwWgt</zorder>
  <zorder>PtzWgt</zorder>
  <zorder>BtnZoomAdd</zorder>
  <zorder>btnZoomSub</zorder>
  <zorder>BtnFocusAdd</zorder>
  <zorder>BtnFocusSub</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
